src/vm/vm.ts

```diff
@@ -429,11 +441,113 @@ function step(n: number = 0): void {
 }
 
 function runUntilPc(): void {
-  let i: number = getLastActiveRowIndex(`${pcColumn}`) - 2;
+  let i: number = getLastActiveRowNumber(`${pcColumn}`, runSheet) - 2;
   let pc: string = runSheet.getRange(`${pcColumn}${i + 1 + 1}`).getValue();
-  while (!(pc === FINAL_PC)) {
+  while (!isCell(pc)) {
```

Comment:
actually this is not really accurate. PC is most of the time a relocatable to segment 0 and currently we've just considered this as always true and consequently stored only the offset. However, PC can be `jmp` to any other segment during the execution.

You can circumvent this limitation by looking for the `final_pc` column (match on the given string that you put in a constant) and they assert again on the final pc